### DISCUSSION TITLE
Flatgeobuf Large Example for Leaflet : add Layer to Layer.Control

### DISCUSSION THREAD

**Discussion by LukasSchefczyk** [flatgeobuf/flatgeobuf#371] [Q&A]
How can make the Example of the Large Flatgeobuf for Leaflet example an Layer to switch it on and off ? i can't wrap my head around the async thing. The Example adds the L.Geojson for each feature to nextresults and add the layergroup nextresults to the map . how can i get it to work with layer.control so i can switch it on and off?

**Comment by bjornharrtell** [comment]
You should probably ask in Leaflet community.
